src/lib/classifyquote.tsx

```diff
@@ -13,15 +13,21 @@ export interface Quote {
 
 /*
 [9:06 PM] Wheatless: wow gj gabe
-[9:06 PM] PJ: [tiff snaps into the sunset
+[9:06 PM] PJ: [tiff snaps into the sunset]
+[2:03 PM] adjective: TASD
+SADA
+AS
+DAS
+DASD
 */
-const discordRegex = /(^\[\d{1,2}:\d{2} [AP]M\] ([^:]*?[^\s]): (.+?)$\n*)+/m;
+const discordHeaderRegex =
```

Comment:
Are you using an 80-column rule or something? I prefer like 100 or 120. Our screens are wide enough.